docs/articles/nunit-engine/extensions/Index.md

```diff
@@ -2,15 +2,10 @@
 uid: engineextensionsindex
 ---
 
-# Engine Extensions
+# NUnit Engine Extensions
```

Comment:
What's the thinking around adding NUnit to the title here? I was thinking since the site is clearly dedicated to NUnit, mentioning it in titles would be redundant. Can you give some insight?

Reply:
Engine is too generic IMO. It's actually the NUnit Test Engine, so I suppose Test Engine would be reasonable as well.

Reply:
Ah, I see. Hadn't considered that; I'm fine with it staying. 👍